server.go

```diff
@@ -34,8 +34,11 @@ var termTmpl = template.Must(template.New("termTmpl").Parse(strings.Replace(`
     |  ssh public keys and Ben (benjojo.co.uk) grabbed them all.          |
     |                                                                     |
     |  That's pretty handy at times :) for example your key is at         |
-    |  https://github.com/{{ .User }}.keys
+    |  https://github.com/{{ .User }}.keys                                |
```

Comment:
That last `|` won't be aligned in the actual output, as `{{.User}}` is of variable length. I guess that's why it was left out ;-)

Reply:
Fixed. Thank you!